_scala3-reference/contextual/derivation.md

```diff
@@ -184,7 +184,7 @@ a `Mirror[T]`. Here is a possible implementation,
 import scala.deriving.Mirror
 
 inline given derived[T](using m: Mirror.Of[T]): Eq[T] =
-  val elemInstances = summonAll[m.MirroredElemTypes]           // (1)
+  lazy val elemInstances = summonAll[m.MirroredElemTypes]           // (1)
```

Comment:
We need `elemInstances` in each branch exactly one, this would add unnecessary overhead?